burr/core/application.py

```diff
@@ -1041,6 +1078,23 @@ def __init__(self):
         self.actions: Optional[List[Action]] = None
         self.start: Optional[str] = None
         self.lifecycle_adapters: List[LifecycleAdapter] = list()
+        self.app_id: str = str(uuid.uuid4())
+        self.partition_key: str = None
+        self.sequence_id: int = 0
+        self.persister = None
+        self.use_entrypoint_from_save_state: bool = None
+        self.default_state: dict = None
+
+    def with_identifiers(
+        self, app_id: str = None, partition_key: str = None, sequence_id: int = None
+    ) -> "ApplicationBuilder":
+        if app_id is not None:
+            self.app_id = app_id
+        if partition_key is not None:
+            self.partition_key = partition_key
+        if sequence_id is not None:
```

Comment:
This is not an identifier? This is a specific aspect of the state -- I don't really see why a user would be setting it? Either:

1. The persister would be
2. It would start at zero